README.md

```diff
@@ -84,18 +98,15 @@ This tool requires a [GitHub personal access token](https://help.github.com/arti
 
 1.  Install [nvm](https://github.com/creationix/nvm) and use the correct node version
 
-        nvm use
+        `nvm use`
```

Comment:
Don't think you need the extra back ticks. It renders as inline code with back ticks otherwise.

```suggestion
        nvm use
```